docs/Users-Guide.md

```diff
@@ -202,6 +203,12 @@ It will also add information about:
 
 - Known CDN providers (based on Timesketch config)
 
+#### IOC Extraction Analyzer
+
+The IOC extraction Analyzer check IOC from database (json) in all fieldnames content specified. Config file is "ioc_extract.yaml".
+
+You can use before "Feature Extraction Analyzer" to extract pattern (ip, url, hash, ...) in specified fields.
```

Comment:
what does `you can use before "Feat..." to extract pattern?` mean? I'm not quite sure, can you elaborate?

Also, as this is written it is only domain names ATM.